scipy/fftpack/_helper.py

```diff
@@ -1,7 +1,10 @@
 import operator
-from numpy.fft.helper import fftshift, ifftshift, fftfreq
+
 import scipy.fft._pocketfft.helper as _helper
+from scipy.fft._helper import fftshift, ifftshift, fftfreq
```

Comment:
This changes the import from numpy to scipy, and post-#19005, that means going through the array API (see [here](https://github.com/scipy/scipy/blob/main/scipy/fft/_helper.py)).